script.js

```diff
@@ -0,0 +1,49 @@
+var clickMe = document.querySelector('#first_paragraph');
+clickMe.addEventListener('click', function (){
+  console.log('h1 got clicked !!!!');
+  document.body.style.background = "blue";
+})
```

Comment:
This is awesome progress! It's cool to see you continuing on. The number of things you can do to a page are almost endless. Here we're changing the background to blue.